webview-ui/src/components/MessageBlock.tsx

```diff
@@ -108,3 +128,18 @@ function getThemedSyntaxStyle(): {
   }
   return syntaxStyle;
 }
+
+const DeleteMessageBlockButton = ({
+  deleteMessageBlock,
+}: {
+  deleteMessageBlock: () => void;
+}) => (
+  <VSCodeButton
+    appearance="icon"
+    aria-label="Remove code block"
+    title="Remove code block"
```

Comment:
The label/title need to be accurate for both text and code blocks

Reply:
Good catch. Thanks!